examples/mappings/lib.rs

```diff
@@ -0,0 +1,281 @@
+
+#[ink::contract]
+mod mappings {
+    use ink::storage::Mapping;
+
+    /// A simple ERC-20 contract.
+    #[ink(storage)]
+    #[derive(Default)]
+    pub struct Mappings {
+        /// Mapping from owner to number of owned token.
+        balances: Mapping<AccountId, Balance>,
+    }
+
+    impl Mappings {
+        /// Demonstrates the usage of `Mappings::default()`
+        ///
+        /// Creates an empty mapping between accounts and balances.
+        #[ink(constructor)]
+        pub fn new() -> Self {
+            let balances = Mapping::default();
+            Self { balances }
+        }
+
+        /// Demonstrates the usage of `Mapping::get()`.
+        /// Returns the balance of a account.
+        ///
+        /// Returns `None` if the account is non-existent.
+        #[ink(message)]
+        pub fn get_balance(&self) -> Option<Balance> {
+            let caller = Self::env().caller();
+            self.balances.get(caller)
+        }
+
+        /// Demonstrates the usage of `Mappings::insert()`.
+        ///
+        /// Assigns the value to a given account.
+        /// Returns the size of the pre-existing balance at the specified key if any.
+        ///
+        /// Returns `None` if the account was non-existent.
+        #[ink(message)]
+        pub fn insert_balance(&mut self, value: Balance) -> Option<u32> {
+            let caller = Self::env().caller();
+            self.balances.insert(caller, &value)
+        }
+
+        /// Demonstrates the usage of `Mappings::size()`.
+        ///
+        /// Returns the size of the pre-existing value at the specified key if any.
+        ///
+        /// Returns `None` if the account was non-existent.
+        #[ink(message)]
+        pub fn size_balance(&mut self) -> Option<u32> {
+            let caller = Self::env().caller();
+            self.balances.size(caller)
+        }
+
+        /// Demonstrates the usage of `Mapping::contains()`.
+        ///
+        /// Returns `true` if the account has any balance assigned to it.
+        #[ink(message)]
+        pub fn contains_balance(&self) -> bool {
+            let caller = Self::env().caller();
+            self.balances.contains(caller)
+        }
+
+        /// Demonstrates the usage of `Mappings::remove()`.
+        ///
+        /// Removes the balance entry for a given account.
+        #[ink(message)]
+        pub fn remove_balance(&mut self) {
+            let caller = Self::env().caller();
+            self.balances.remove(caller);
+        }
+
+        /// Demonstrates the usage of `Mappings::take()`.
+        ///
+        /// Returns the balance of a given account
+        /// removing it from storage.
+        ///
+        /// Returns `0` if the account is non-existent.
+        #[ink(message)]
+        pub fn take_balance(&mut self) -> Option<Balance> {
+            let caller = Self::env().caller();
+            self.balances.take(caller)
+        }
+    }
+
+    #[cfg(test)]
```

Comment:
```suggestion
        #[cfg(all(test, feature = "e2e-tests"))]
```

We now have these feature flagged since we don't want people to need a node running all the time just to run tests (see `lang-err-integration-tests`). Can you also add the feature to the contract's `Cargo.toml`?